assets/chef-recipes/cookbooks/mdbe_build/recipes/default.rb

```diff
@@ -86,40 +84,68 @@
 ]
 
 debian_packages = %w[
-  default-jdk
```

Comment:
This package was moved to each of version-specific lists for each Debian box. Maybe it is better to keep it here?